lib/frrlua.h

```diff
@@ -162,10 +162,12 @@ void lua_decode_stringp(lua_State *L, int idx, char *str);
 void *lua_tostringp(lua_State *L, int idx);
 
 /*
- * No-op decocder
+ * No-op decocders
```

Comment:
*decoders

Reply:
fixed in https://github.com/FRRouting/frr/pull/8982/commits/6d28552d2da20852a3580da17894d7b7db6756ba